community/libev/build

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh -e
+
+# export V=1 for verbose output when building
+[ "${V:-0}" -eq 1 ] > /dev/null 2>&1 && set -x || :
+
+PATCHES="${PATCHES:-}"
+
+PATCHES="
+libev-4.25-pc.patch
+${PATCHES}
+"
+
+if [ -n "${PATCHES}" ]
+then
+	for patch in ${PATCHES}
+	do
+		if [ -z "${patch}" ] || [ ! -e "${patch}" ]
+		then
+			continue
+		fi
+		patch -p1 < "${patch}"
+	done
+fi
+
+sed -i -e "/^include_HEADERS/s/ event.h//" Makefile.am
+
+autoreconf -vfi
```

Comment:
Is `autoreconf` necessary (also the `sed` above it)?

Reply:
The `sed` is needed to prevent a conflict with libevent (both install `event.h`). I'll remove the `autoreconf`.

Reply:
Is this not a simpler solution?

```sh
# Goes after 'make install'.
rm -f "$1/usr/include/event.h"
```